Front_end/lib/pages/user_authentication.dart

```diff
@@ -0,0 +1,193 @@
+    var url = Uri.parse('https://10.0.2.2:5001/clientes');
+    var clientes = List<Cliente>.empty(growable: true);
+    var response = await http.get(url);
+    var clienteList = json.decode(response.body);
+    for (Map<String, dynamic> cliente in clienteList) {
+      clientes.add(Cliente.fromJson(cliente));
+    }
+    return clientes;
+  }
+
+  Future<Pedido> getCarrinhoCliente() async {
+    var url = Uri.parse('https://10.0.2.2:5001/clientes/${idCliente}');
+    var response = await http.get(url);
+    var pedido = json.decode(response.body);
+    return Pedido.fromJson(pedido);
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    getCliente().then((var clientes) => setState(() {
+          _clientes = clientes;
+        }));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: Colors.white,
+        body: Column(
+
+            // Widgets organizados em uma coluna
+            children: [
+              // Texto "Lojinha"
+              Expanded(
+                  flex: 3,
+                  child: Container(
+                      alignment: Alignment.bottomCenter,
+                      child: Padding(
+                        padding: const EdgeInsets.all(50.0),
+                        child: Text("Lojinha",
+                            style: TextStyle(
+                              color: Theme.of(context).accentColor,
+                              fontSize: 50,
+                              fontWeight: FontWeight.w600,
+                            )),
+                      ))),
+
+              // Coluna que armazenada o Texto e o Dropdown
+              Expanded(
+                  flex: 1,
+                  child: Container(
+                      alignment: Alignment.center,
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        children: [
+                          Text("Selecione o usuário",
+                              style:
+                                  TextStyle(color: Colors.black, fontSize: 23),
+                              textAlign: TextAlign.left),
+                          // DropdownButton
+                          Padding(
+                            padding: EdgeInsets.fromLTRB(100, 0, 100, 0),
+                            child: DropdownButton(
+                              hint: Text(
+                                valorString,
+                                style: TextStyle(
+                                    fontSize: 18, color: Colors.black),
+                              ),
+                              icon: const Icon(Icons.arrow_drop_down),
+                              iconSize: 24,
+                              elevation: 16,
+                              isExpanded: true,
+                              style: TextStyle(
+                                  color: Colors.black), // Cor do texto
```

Comment:
Preto não está nas cores do theme, se for usar é necessario acrescentar a cor no theme e chamar pelo código.